app.ts

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
check this style guide, for better code readability: https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines